timescaledb/how-to-guides/connecting/azure-data-studio.md

```diff
@@ -1,15 +1,124 @@
 keywords: [connect]
 ---
 
-# Azure Data Studio
-Microsoft Azure Data Studio is a cross-platform database tool that is available
-for Microsoft Windows, Apple macOS, or various Linux flavors. You can try Azure
-Data Studio for free.
+# Connect to TimescaleDB using Azure Data Studio
 
-For more information about Azure Data Studio, including download and
-installation instructions, see [azure.microsoft.com][ms-azure-data-studio].
+Azure Data Studio is a cross-platform database tool for data professionals using
+on-premises and cloud data platforms on Windows, macOS, and Linux. You can
+connect to TimescaleDB hosted on your local machine or on a remote server. You
+can also connect to Timescale Cloud service and Managed Service for TimescaleDB
+service.
 
-[ms-azure-data-studio]: https://azure.microsoft.com/en-us/services/developer-tools/data-studio/
+## Before you begin
+
+*   Download and install [Azure Data Studio][ms-azure-data-studio].
+*   Install the [PostgreSQL extension for Azure Data Studio][postgresql-azure-data-studio].
+
+### Connecting to TimescaleDB hosted locally, or on a remote server
+
+<procedure>
+
+1.  Start `Azure Data Studio`.
+1.  Click the `New Connection` icon in the `SERVERS`page.
+1.  In the form that pops up, go to `Connection type` and select `PostgreSQL`
+    from the drop-down. Fill in the fields using the server name, user name, and
+    password for your TimescaleDB server. For description of the settings in
+    `Connection Details`, see [Connection Details][connection-details]
+
+   <img class="maincontent__illustration"src="https://s3.amazonaws.com/assets.timescale.com/docs/images/local_remote_connection_screen.png"alt="Connection screen for local or remote timescaledb"/>
+
+1.  Click `Advanced`.
+1.  Type the port number in the `Port` field and click `OK`.
+1.  Click `Connect`.
+
+After successfully connecting, your server opens in the `SERVERS`sidebar.
+
+</procedure>
+
+### Connecting to Timescale Cloud service
+
+<procedure>
+
+1.  Sign in to the [Timescale Cloud portal][tsc-portal].
+1.  In the `Services` tab, find the service you want to connect to, and check
+    it is marked as `Running`.
+1.  Click the name of the service you want to connect to see the connection
+    information. Take a note of the `Host`, `Port`, `Database name`, and `Username`.
+1.  Navigate to the `Operations` tab, and click `Reset password`. You can choose
+    your own password for the service, or allow Timescale Cloud to generate a
+    secure password for you. Take a note of your new password.
+1.  Start `Azure Data Studio`.
+1.  Click the `New Connection` icon in the `SERVERS`page.
+1.  In the form that pops up, go to `Connection type`and select `PostgreSQL`from
+    the drop-down. Fill in the fields using the server name, user name, port, and
+    password for your TimescaleDB Cloud service. For description of the settings
+    in `Connection Details`, see [Connection Details][connection-details].
```

Comment:
```suggestion
1.  In the `FIXME` dialog, navigate to `Connection type`and select `PostgreSQL`. 
    Complete the form using the server name, user name, port, and password 
    for your Timescale Cloud service.
```

Reply:
Considering the procedure starts by telling them how to retrieve those details, we probably don't need to provide a link to that info. Alternatively, of course, you could include the link, but don't bother walking them through doing it first. Up to you.

Reply:
@Loquacity what do you mean by _`FIXME` dialog_?
![Selection_092](https://user-images.githubusercontent.com/10092181/184129293-32a3182f-574f-40a2-897a-a7ee96d11d2e.png)

Reply:
Maybe add something to the `server name` field to indicate it's the same thing as `host`.
`You can find the server name information as "host" in Timescale Cloud.` or something like this

Reply:
also "Database field" needs to be filled out as well (it's always `tsdb` for TS Cloud)